src/pages/attendees/summit-attendees-list-page.js

```diff
@@ -353,6 +361,18 @@ class SummitAttendeeListPage extends React.Component {
                             style={{ width: "100%", height:40, color: '#337ab7', fontSize: '10px' }}
                         />
                     </div>
+                    <div className="col-md-4">
+                        <SegmentedControl
+                            name="checkedInFilter"
+                            options={[
+                                { label: "All", value: null, default: checkedInFilter === null},
+                                { label: "Checked in", value: "CHECKED_IN", default: checkedInFilter === "HAS_TICKETS" },
```

Comment:
i do think that here  checkedInFilter === "HAS_TICKETS"  should be
 checkedInFilter === "CHECKED_IN"